src/Microsoft.Diagnostics.EventFlow.Inputs.Etw/MsgPackTraceEventSession.cs

```diff
@@ -0,0 +1,166 @@
+﻿using Microsoft.Diagnostics.Tracing;
```

Comment:
Yes. I'll add the same copyright as the rest of code, as I'm acting as Microsoft Employee w.r.t. this commit :) I am not 100% sure if this input adapter needs to live in the mainline, or moved elsewhere. It would be great if it is in the mainline, assuming that adding dependency on `MsgPack/CLI` package is tolerable.

Reply:
Yes, taking a dependency on `MsgPack/CLI` is fine, we have other parts of EventFlow that depend on Apache-licensed NuGet packages, Elasticsearch for example.